src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/WorkspaceProjectStateChangeDetector.cs

```diff
@@ -184,6 +197,8 @@ private async Task UpdateAfterDelay(ProjectId projectId)
         {
             await Task.Delay(EnqueueDelay);
 
+            OnStartingDelayedUpdate();
```

Comment:
Which way does the notification need to flow? This allows the worker to wait until the test is ready for it to proceed.

Reply:
Discussed offline.